src/PointwiseFunctions/AnalyticSolutions/Xcts/ConstantDensityStar.hpp

```diff
@@ -0,0 +1,172 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"     // IWYU pragma: keep
+#include "Elliptic/Systems/Xcts/Tags.hpp"       // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Xcts {
+namespace Solutions {
+
+/*!
+ * \brief A constant density star in general relativity
+ *
+ * \detail This solution describes a star with constant density \f$\rho_0\f$
+ * that extends to a (conformal) radius \f$R\f$. It solves the XCTS Hamiltonian
+ * constraint that reduces to the non-linear elliptic equation \f[
+ * \Delta^2\psi+2\pi\rho\psi^5=0
+ * \f] for the conformal factor \f$\psi\f$ (see `Xcts`) under the following
+ * assumptions \cite Baumgarte2006ug :
+ * - Time-symmetry \f$K_{ij}=0\f$
+ * - Conformal flatness \f$\overline{\gamma}=\delta\f$, so \f$\Delta\f$ is the
+ * flat-space Laplacian
+ * - Spherical symmetry
+ * Imposing boundary conditions \f[
+ * \frac{\partial\psi}{\partial r}=0 \quad \text{for} \quad r=0\\
+ * \psi\rightarrow 1 \quad \text{for} \quad r\rightarrow\infty
+ * \f] and considering the energy density \f[
+ * \rho(r\leq R)=\rho_0 \quad \text{and} \quad \rho(r>R)=0
+ * \f] of the star the authors of \cite Baumgarte2006ug find the solution \f[
+ * \psi(r\leq R)=C u_\alpha(r) \quad \text{and}
+ * \quad \psi(r>R)=\frac{\beta}{r} + 1
+ * \f] with \f$C=(2\pi\rho_0/3)^{-1/4}\f$, the Sobolev functions \f[
+ * u_\alpha(r)=\frac{(\alpha R)^{1/2}}{\left(r^2+(\alpha R)^2\right)^{1/2}}
+ * \f] and real parameters \f$\alpha\f$ and \f$\beta\f$ that are determined by
+ * the following relations: \f[
+ * \rho_0 R^2=\frac{3}{2\pi}f^2(\alpha) \quad \text{with}
+ * \quad f(\alpha)=\frac{\alpha^5}{(1+\alpha^2)^3} \\
+ * \frac{\beta}{R} + 1 = C u_\alpha(R)
+ * \f]
+ *
+ * This solution is described in detail in \cite Baumgarte2006ug since it
+ * exhibits the non-uniqueness properties that are typical for the XCTS system.
+ * In the simple case of the constant-density star the non-uniqueness is
+ * apparent from the function \f$f(\alpha)\f$, which has two solutions for any
+ * \f$\rho_0\f$ smaller than a critical density \f[
+ * \rho_\mathrm{crit}=\frac{3}{2\pi R^2}\frac{5^2}{6^6}
+ * \approx\frac{0.0320}{R^2}
+ * \f], a unique solution for \f$\rho_0=\rho_\mathrm{crit}\f$ and no solutions
```

Comment:
move the comma into the equation (currently as rendered, the comma begins the next line)